packages/eas-cli/src/project/projectUtils.ts

```diff
@@ -1,6 +1,13 @@
-import { getConfig } from '@expo/config';
+import { getConfig, getConfigFilePaths } from '@expo/config';
```

Comment:
Ignore this file. It will be rebased away once this PR lands: https://github.com/expo/eas-cli/pull/107